### PR TITLE
Load libor publisher addr from a config file

### PR DESCRIPTION
**Pull request by feihujiang**
Signed-off-by: feihujiang <jiangfeihu@huawei.com>

@jjason Could you please have a look?

### CODE REVIEW

File: extensions/bond/sawtooth_bond/txn_family.py [hunk 1]
```diff
@@ -49,6 +50,19 @@
...
+    try:
+        with open('./etc/sawtooth-bond.json') \
+                as bond_fd:
+            bond_config = json.load(bond_fd)
+    except IOError, ex:
+        LOGGER.exception('IOError: %s', str(ex))
+        return
+
+    libor_publisher_addr = bond_config["LiborPublisherAddr"]
+    CreateLIBORUpdate.LIBOR_PUBLISHER_ADDR(libor_publisher_addr)
```
Comment (jjason) [inline extensions/bond/sawtooth_bond/txn_family.py]: Am I missing something?  LIBOR_PUBLISHER_ADDR has been been made a global variable in sawtooth_bond.updates.libor and here you are accessing it like a function on the CreateLIBORUpdate class.

File: extensions/bond/sawtooth_bond/txn_family.py [hunk 2]
```diff
@@ -49,6 +50,19 @@
...
+def _load_config(journal):
+    try:
+        with open('./etc/sawtooth-bond.json') \
+                as bond_fd:
+            bond_config = json.load(bond_fd)
+    except IOError, ex:
+        LOGGER.exception('IOError: %s', str(ex))
+        return
+
+    libor_publisher_addr = bond_config["LiborPublisherAddr"]
```
Comment (jjason) [inline extensions/bond/sawtooth_bond/txn_family.py]: I think you need to guard against the "LiborPublisherAddr" attribute not existing in the configuration file:
libor_publisher_addr = bond_config.get('LiborPublisherAddr')
if libor_publisher_addr is not None:
  ....

This way the code works with the default LIBOR address if one is not provided.  Either that, or flag an error if it doesn't exist.

**Comment by feihujiang** [comment]
@jjason Sorry, I forget to say this PR is just an idea. The idea in this PR is accepted, isn't it? If it is, I'll update this PR tomorrow.

**Comment by jjason** [comment]
@vaporos - I tend to agree.

**Pull request closed without merging**